app-emulation/playonlinux/playonlinux-4.3.4.ebuild

```diff
@@ -0,0 +1,106 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python2_7 )
+
+inherit desktop eutils python-single-r1 xdg
+
+MY_PN="PlayOnLinux"
+
+DESCRIPTION="Set of scripts to easily install and use Windows games and software"
+HOMEPAGE="https://playonlinux.com/"
+SRC_URI="https://www.playonlinux.com/script_files/${MY_PN}/${PV}/${MY_PN}_${PV}.tar.gz"
+
+LICENSE="GPL-3"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="winbind"
+REQUIRED_USE="${PYTHON_REQUIRED_USE}"
+
+DEPEND=""
+RDEPEND="${PYTHON_DEPS}
+	app-arch/cabextract
+	app-arch/p7zip
+	app-arch/unzip
+	app-crypt/gnupg
+	app-misc/jq
+	dev-python/wxpython:3.0[${PYTHON_USEDEP}]
+	media-gfx/icoutils
+	|| ( net-analyzer/netcat net-analyzer/openbsd-netcat )
+	net-misc/wget
+	virtual/imagemagick-tools
+	virtual/wine
+	winbind? ( net-fs/samba[winbind] )
```

Comment:
When sorting these dependency lists, remember that unconditional deps come before conditional deps. And in lexicographic ordering the `||` is probably above everything else...

Reply:
What makes you say so? I do not see any specific instructions or recommendations on this neither in devmanual nor in skel.ebuild.
The examples from https://devmanual.gentoo.org/quickstart/index.html and https://devmanual.gentoo.org/general-concepts/dependencies/ do not follow your recommendation. I would be grateful for a citation.

In this case I would like to leave it as it is until maintainter's comment. Or a developer that is going to push this commit can reorder the dependencies in a way he likes.

Reply:
I'm not totally sure what juippis is talking about to be honest, I personally prefer it in lexigraphical order which it seems to be.

Reply:
Well it's something that's always asked to "fix" by proxy-maint devs, 
https://github.com/gentoo/gentoo/pull/9045#discussion_r200806099
so I guess it just got stuck with me. Could be styling prefence in the end. 

Seems like I was wrong about the position of `||` in lexicographic ordering. It doesn't come first, it comes last. 
```
$ cat tmp1.txt 
${PYTHON_DEPS}
app-arch/cabextract
app-arch/p7zip
app-arch/unzip
app-crypt/gnupg
app-misc/jq
dev-python/wxpython:3.0[${PYTHON_USEDEP}]
media-gfx/icoutils
|| ( net-analyzer/netcat net-analyzer/openbsd-netcat )
net-misc/wget
virtual/imagemagick-tools
virtual/wine
winbind? ( net-fs/samba[winbind] )
x11-apps/mesa-progs
x11-terms/xterm

$ LC_ALL=C sort tmp1.txt 
${PYTHON_DEPS}
app-arch/cabextract
app-arch/p7zip
app-arch/unzip
app-crypt/gnupg
app-misc/jq
dev-python/wxpython:3.0[${PYTHON_USEDEP}]
media-gfx/icoutils
net-misc/wget
virtual/imagemagick-tools
virtual/wine
winbind? ( net-fs/samba[winbind] )
x11-apps/mesa-progs
x11-terms/xterm
|| ( net-analyzer/netcat net-analyzer/openbsd-netcat )
```